modules/040-control-plane-manager/images/kubernetes-api-proxy/werf.inc.yaml

```diff
@@ -0,0 +1,14 @@
+---
+image: {{ $.ModuleName }}/{{ $.ImageName }}
+from: {{ env "BASE_NGINX_ALPINE" }}
+import:
+- image: common/pause
```

Comment:
I assume that pause binary is not required for this image.

Reply:
yep, thanks !